backend/src/modules/user/user.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { UserService } from './user.service';
+
+describe('UserService', () => {
+  let service: UserService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [UserService],
```

Comment:
Man atrodo tu blogai kazkaip exportuoji ir UserService ir UserController ir del to testai failina. Reikia pasibandyt lokaliai